relayer/pkg/chainlink/ocr2/config_digester.go

```diff
@@ -48,6 +48,10 @@ func (d offchainConfigDigester) ConfigDigest(cfg types.ContractConfig) (types.Co
 		return configDigest, errors.New("must have equal number of signers and transmitters")
 	}
 
+	if len(cfg.Signers) == 0 {
+		return configDigest, errors.New("number of signers and transmitters shall be greater than 0")
+	}
```

Comment:
We should actually check something like:

```suggestion
	if len(cfg.Signers) <= 3 * cfg.F {
		return configDigest, errors.New("number of oracles must be greater than 3*f")
	}
```

This is what the contract also checks [here](https://github.com/smartcontractkit/chainlink-starknet/blob/3df46ca75401919aee6f9b65675ec48792a614c1/contracts/src/chainlink/cairo/ocr2/aggregator.cairo#L228).